build.bash

```diff
@@ -8,48 +8,48 @@
 #
 # Outputs compiled program in current directory.
 # Default file name is 'ecaddy'.
-#
+
 set -e
 
-output="$1"
-if [ -z "$output" ]; then
-	output="ecaddy"
-fi
+: ${output_filename:="$1"}
+: ${output_filename:="ecaddy"}
 
 pkg=main
+ldflags=()
 
 # Timestamp of build
-builddate_id=$pkg.buildDate
-builddate=`date -u`
+name="${pkg}.buildDate"
+value=$(date --utc +"%F %H:%M:%SZ")
+ldflags+=("-X" "\"${name}=${value}\"")
```

Comment:
But I actually prefer the old/current date format better... can we keep it, pretty plz? :smiley: